doc/connectivity/connectome_extraction.rst

```diff
@@ -28,9 +28,10 @@ Connectome extraction: inverse covariance for direct connections
 Sparse inverse covariance for functional connectomes
 =====================================================
 
-Resting-state functional connectivity can be obtained by estimating a
-covariance (or correlation) matrix for signals from different brain
-regions. The same information can be represented as a weighted graph,
+Functional connectivity can be obtained by estimating a covariance
+(or correlation) matrix for signals from different brain
+regions decomposed, for example on resting-state or movie-watching datasets.
```

Comment:
Light suggestion to generalize to all naturalistic stimuli (e.g., story-listening, etc)
```suggestion
regions decomposed, for example on resting-state or naturalistic-stimuli datasets.
```